homeassistant/components/alarm_control_panel/__init__.py

```diff
@@ -94,6 +96,18 @@ def alarm_arm_night(hass, code=None, entity_id=None):
     hass.services.call(DOMAIN, SERVICE_ALARM_ARM_NIGHT, data)
 
 
+@bind_hass
+def alarm_arm_max(hass, code=None, entity_id=None):
```

Comment:
New base component services needs to be approved via an issue in our architecture repo.

https://developers.home-assistant.io/docs/en/entity_index.html#changing-the-entity-model